locales/English/user.php

```diff
@@ -72,29 +72,13 @@
 	}
 }
 
-foreach ($User->options->value as $name=>$val) 
-{
-	print '<div><li class="settings"><strong>'.UserOptions::$titles[$name].':</strong></li>';
-	foreach (UserOptions::$possibleValues[$name] as $possible) 
-	{
-		print ' <input type="radio" name="userForm['.$name.']" value="'.$possible.'" '. ($val == $possible ? 'checked' :'') . ' > '. $possible;
-	}
-	print '</div></br>';
-}
- 	                               
-if( $User->type['User'] ) 
-{
-	if( isset(Config::$enabledOptInFeatures) && Config::$enabledOptInFeatures > 0 )
+if( isset(Config::$enabledOptInFeatures) && Config::$enabledOptInFeatures > 0 )

```

Comment:
Using the ternary operator we could simplify this line so that it isn't so long, this is just a recommendation. Also, we should only explicitly show this the value of `1`.

`(Config::$enabledOptInFeatures ?? 0) === 1`.